test/packages/sql_input/manifest.yml

```diff
@@ -0,0 +1,67 @@
+format_version: 1.0.0
+name: sql_input
+title: SQL Input
+description: >-
+  Execute custom queries against an SQL database and store the results in Elasticsearch.
+type: input
+version: 1.0.1
+release: ga
+license: basic
+categories:
+  - datastore
+policy_templates:
+  - name: sql_query
+    type: metrics
+    title: SQL Query
+    description: Query the database to capture metrics.
+    input: sql
+    template_path: input.yml.hbs
+    vars:
+      - name: hosts
+        type: text
+        title: Hosts
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - http://127.0.0.1
+      - name: period
+        type: text
+        title: Period
+        multi: false
+        required: true
+        show_user: true
+        default: 10s
+      - name: driver
+        type: text
+        title: Driver
+        description: "Supported database drivers: mssql, mysql, oracle, postgres"
+        multi: false
+        required: true
+        show_user: true
+        default: "mysql"
+      - name: sql_query
+        type: text
+        title: Query
+        multi: false
+        required: true
+        show_user: true
+        default: "SHOW GLOBAL STATUS LIKE 'Innodb_system%'"
+      - name: sql_response_format
+        type: text
+        title: Response format
+        description: "Supported response formats: variables, table"
+        multi: false
+        required: true
+        show_user: false
+        default: "variables"
```

Comment:
For such cases, it would be great to have a pattern or allowed values. cc @joshdover

Reply:
I played with patterns (as regexp) here https://github.com/elastic/package-spec/pull/245. I would prefer a list of allowed values :)